tasks/Minimum-Effort Tacit Coordination Game.md

```diff
@@ -35,7 +40,21 @@ Treatment C: In treatment C, group size was reduced to two people who were stran
 
 
 ## Instructions
-Authors no longer have instructions 
```

Comment:
So, to be clear, these are from one of the documents linked above, right?

Reply:
Correct- the authors included this source in the footnotes of the main paper- I missed that the first time around.